packages/near-lake-primitives/src/types/receipts.ts

```diff
@@ -167,7 +169,7 @@ class DeleteAccount {
 export type Operation =
   | 'CreateAccount'
   | DeployContract
-  | FunctionCall
```

Comment:
Was this incorrect? Is this a breaking change?

Reply:
@pavelisnear This file is where I ran into the Operations types not matching the data. I only changed FunctionCall but they should probably all be corrected.